### DISCUSSION TITLE
Load Order Management

### DISCUSSION THREAD

**Discussion by AtomicTEM** [NicholasDJM/ProvisionsModManager#7] [Ideas]
Do you plan on having a simple way of managing the load order of mods? Like you could have a vertical list which would sorted by the alphabetical order of the Folders and VPKs of a mods archive, and when you move the mods in the list up or down it will append the mods files with like a Z or A or # t ochange its load order position. I think this combines with somethin like your vkpscan would be amazing.

**Comment by NicholasDJM** [comment]
Yes, managing the load order of your mods will be a core feature. VPKScan is an initial proof of concept for Provisions Mod Manager, and it's feature of detecting conflicts is also a core feature.

As for the naming of files, I've decided to combine all mods into a single VPK archive for performance reasons. If you've installed a lot of mods, and found the text in game is just placeholders, it's because Team Fortress 2 fails to load the localization files. By combining all mod files into a single archive, TF2, will only need to load single file, or at least very few.

And finally, the load order itself will be simple numbering system. The mods will be listed by default by which mods you want to load first. You'll be able to change the order via drag and drop, up and down buttons, and changing the number directly.
